guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -35,13 +35,14 @@ ifndef::satellite[]
 ----
 # {package-install} foreman-proxy-content
 ----
+endif::[]
```

Comment:
Nice job spotting this!

Reply:
I also found this in https://github.com/theforeman/foreman-documentation/pull/1006 along with more things. I get the impression we don't really read the guides when reviewing. I also noticed many URLs are printed now instead of a nice label.

Reply:
I also spotted that we have some `xref:#xxx` when the hash should not be really required, it might even break things, but ccutil did not complain. Did not fix it yet tho.